pkg/registry/rbac/validation/rule.go

```diff
@@ -176,11 +177,14 @@ func (d *roleBindingDescriber) String() string {
 }
 
 func (r *DefaultRuleResolver) VisitRulesFor(user user.Info, namespace string, visitor func(source fmt.Stringer, rule *rbacv1.PolicyRule, err error) bool) {
+
+	userTenant := user.GetTenant()
```

Comment:
Do we need to handle the case that userTenant is empty?

Reply:
userTenant being empty sounds to me like coming into border checkpoint without a valid passport.